run.sh

```diff
@@ -205,16 +205,24 @@ if [ ! -f ./gerobug_dashboard/secrets/gerobug_secret.env ]; then
     echo 'DJANGO_SUPERUSER_PASSWORD="'$DJANGO_SUPERUSER_PASSWORD'"' > ./gerobug_dashboard/secrets/gerobug_secret.env
 fi
 
+function docker_cmd() {
+    if docker --version | grep -q "version 2[0-9]\."; then
+        echo "docker compose"
+    else
+        echo "docker-compose"
+    fi
+}
```

Comment:
I think it might be worth coming up with a better way to detect which one to use, and I'm open to suggestions - this was an easy way to do it `(but of course will fail if/when 3x.x.x is released)`